third_party/pocketfft/BUILD.bazel

```diff
@@ -4,8 +4,21 @@ package(default_visibility = ["//visibility:public"])
 
 cc_library(
     name = "pocketfft",
-    hdrs = ["pocketfft_hdronly.h"],
-    copts = ["-fexceptions"],
+    hdrs = ["src/ducc0/fft/fft.h"],
+    srcs = ["src/ducc0/fft/fft1d.h",
+            "src/ducc0/infra/aligned_array.h",
+            "src/ducc0/infra/error_handling.h",
+            "src/ducc0/infra/mav.h",
+            "src/ducc0/infra/simd.h",
+          "src/ducc0/infra/threading.h",
+          "src/ducc0/infra/threading.cc",
+            "src/ducc0/infra/useful_macros.h",
+            "src/ducc0/math/cmplx.h",
+            "src/ducc0/math/unity_roots.h",
+],
+    copts = ["-fexceptions", "-ffast-math"],
     features = ["-use_header_modules"],
+    local_defines = [],
```

Comment:
Unused; remove.

Reply:
Great to see this making progress! I think I made the necessary adjustments.